site/src/docs/patterns/navigation/building-navigation.mdx

```diff
@@ -0,0 +1,522 @@
+
+# Building navigation
+
+<LeadParagraph>
+  Navigation components are crucial as they help users navigate through an application or website seamlessly.
+</LeadParagraph>
+
+## Principles
+
+Adhering to these principles can create an effective and user-friendly navigation component that enhances the overall user experience.
+
+<div
+  class="guideline-do"
+  style="background: var(--color-success-light); padding: var(--spacing-s); margin-bottom: var(--spacing-layout-2-xs); max-width:600px;"
+>
+  <div class="guideline-do-label" style="color:var(--color-success); margin-bottom:var(--spacing-s);">
+    <IconCheck style={{ marginRight: 'var(--spacing-3-xs)', verticalAlign: 'middle' }} size="s" />
+    <strong>Do</strong>
+  </div>
+  <ul style="margin-bottom:0;">
+    <li>
+      <strong>Follow the documentation closely.</strong>
+      <ul style="margin-top:0;">
+        <li>
+          Adhere to the guidelines, best practices, and examples provided in the documentation to ensure proper
+          implementation of the navigation pattern.
+        </li>
+      </ul>
+    </li>
+    <li>
+      <strong>Keep the user in mind.</strong>
+      <ul style="margin-top:0;">
+        <li>
+          Consider the needs, preferences, and expectations of your target audience when implementing the navigation
+          pattern.
+        </li>
+      </ul>
+    </li>
+    <li>
+      <strong>Maintain consistency.</strong>
+      <ul style="margin-top:0;">
+        <li>
+          Keep the design, interaction, and behavior of the navigation pattern consistent across different platforms and
+          devices.
+        </li>
+      </ul>
+    </li>
+    <li>
+      <strong>Test and iterate.</strong>
+      <ul style="margin-top:0;">
+        <li>
+          Regularly test the navigation pattern with real users, gather feedback, and make improvements to ensure it
+          meets user needs and expectations.
+        </li>
+      </ul>
+    </li>
+    <li>
+      <strong>Pay attention to label lengths.</strong>
+      <ul style="margin-top:0;">
+        <li>Keep the as concise as possible.</li>
+      </ul>
+    </li>
+    <li>
+      <strong>Be responsive.</strong>
+      <ul style="margin-top:0;">
+        <li>Make sure the navigation pattern works well on different devices and screen sizes.</li>
+      </ul>
+    </li>
+    <li>
+      <strong>Prioritize important elements.</strong>
+      <ul style="margin-top:0;">
+        <li>
+          Organize the navigation items hierarchically, highlighting the most important elements and grouping similar
+          items together.
+        </li>
+      </ul>
+    </li>
+    <li>
+      <strong>Customize responsibly.</strong>
+      <ul style="margin-top:0;">
+        <li>
+          When customizing the navigation pattern, ensure that the changes do not negatively impact usability,
+          accessibility, or consistency.
+        </li>
+      </ul>
+    </li>
+  </ul>
+</div>
+
```

Comment:
Do: make sure the navigation links correspond the page they open. Also if there are multiple links to the same page, they should have the same label.

Reply:
Do: think the information architecture and site hierarchy. What is most important info, how many levels do you need etc?

Reply:
Good points!